scraper/src/fetch_permapeople.js

```diff
@@ -0,0 +1,54 @@
+import axios from 'axios';
+import fs from 'fs';
+import { parse as json2csv } from 'json2csv';
+import { config } from 'dotenv';
+
+config();
+
+const apiUrl = 'https://permapeople.org/api/plants/';
+const headers = {
+  'x-permapeople-key-id': process.env.PERMAPEOPLE_KEY_ID,
+  'x-permapeople-key-secret': process.env.PERMAPEOPLE_KEY_SECRET,
+};
+
+async function fetchAllPlants() {
+  const allPlants = [];
+  const response = await axios.get(apiUrl, { headers });
+  let plants = response.data.plants;
+  let i = 0;
+  while (plants.length > 0) {
+    // if (i == 1) break;
+    console.log(`Fetching page ${i + 1}`);
+    plants.forEach((plant) => {
+      plant = plant.data.reduce((acc, curr) => {
+        acc[curr.key] = curr.value;
+        return acc;
+      }, plant);
```

Comment:
This flattens the data array of the response out into the plant response object, right?
I think a comment would be nice here and a link to the relevant sections in the API documentation.

Reply:
Yeah, it is coming. Thank you overall for your feedback, great catches indeed!